test/e2e/benchmark/throughput.go

```diff
@@ -30,11 +30,11 @@ func TwoNodeSimple(logger *log.Logger) error {
 		SelfDelegation:     10000000,
 		CelestiaAppVersion: latestVersion,
 		TxClientVersion:    testnet.TxsimVersion,
-		EnableLatency:      true,
+		EnableLatency:      false,
 		LatencyParams:      LatencyParams{100, 10}, // in  milliseconds
-		BlobsPerSeq:        1,
-		BlobSequences:      1,
-		BlobSizes:          "10000-10000",
+		BlobsPerSeq:        6,
+		BlobSequences:      50,
+		BlobSizes:          "200000",
```

Comment:
These value are set high to simulate the situation in which the `ResourceExhausted` error occurs as explained in #3437.